CMakeLists.txt

```diff
@@ -152,9 +152,16 @@ foreach(SD_API_VER ${SD_API_VERS})
     string(TOLOWER ${SD_API_VER} SD_API_VER_L)
     set(CONNECTIVITY_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/hex/${SD_API_VER_L}")
 
-    set(HEX_FILE_GLOB "${CONNECTIVITY_DIRECTORY}/connectivity_${CONNECTIVITY_VERSION}*.hex")
+    set(HEX_FILE_GLOB 
+        "${CONNECTIVITY_DIRECTORY}/connectivity_${CONNECTIVITY_VERSION}*.hex"
+        "${CONNECTIVITY_DIRECTORY}/*_softdevice.hex"
+    )
     set(DFU_FILE_GLOB "${CONNECTIVITY_DIRECTORY}/connectivity_${CONNECTIVITY_VERSION}*.zip")
-    set(LICENSE_FILE_GLOB "${CONNECTIVITY_DIRECTORY}/*license-agreement.txt")
+
+    set(LICENSE_FILE_GLOB 
+        "${CONNECTIVITY_DIRECTORY}/*licence-agreement.txt"
```

Comment:
I think there's no need to change the license glob, or there's a mistake I don't see.

Reply:
There is, they changed the name in the SDv6.1.1 release to licence.

Reply:
Tiny change, I must be dislexic :)